stratum/hal/bin/np4intel/docker/configs/stratum.flags

```diff
@@ -1,10 +1,6 @@
 -define phal_with_onlp=false
--external_stratum_urls=0.0.0.0:28000
--grpc_max_recv_msg_size=256
--persistent_config_dir=/stratum_configs/
 -forwarding_pipeline_configs_file=/stratum_configs/pipeline_config.pb.txt
 -dpdk_config=/stratum_configs/dpdk_config.pb.txt
 -chassis_config_file=/stratum_configs/chassis_config.pb.txt
 -write_req_log_file=/stratum_logs/p4_writes.pb.txt
```

Comment:
Change to default path like `/etc/stratum` and `/var/log/stratum` ?

Reply:
We could and should going forward. But I'm not familiar with the platform so I put of any changes there until we can try them ourself.

Reply:
@craigsdell Please check there changes and add more if you think it also apply to the NP4 platform